pilot/pkg/bootstrap/server.go

```diff
@@ -273,6 +277,21 @@ func NewServer(args *PilotArgs) (*Server, error) {
 		return nil, fmt.Errorf("config validation: %v", err)
 	}
 
+	if dns.DNSAddr.Get() != "" {
+		if err := s.initDNSTLSListener(dns.DNSAddr.Get()); err != nil {
+			log.Warna("Failed to start DNS-over-TLS listener ", err)
```

Comment:
Why do you need dns over tls for local host (coredns to istiod)? It’s excessive isn’t it ?

Reply:
It's the other way around - see my previous comment. Istiod is handling DNS-over-TLS, with this listener. 

When K8S DNS (or whatever the pods are configured to use) is secure - Pods will use that DNS server as resolver, but we still need .global to be delegated to istiod, and that should
also be done over TLS. 

So Istiod will either handle all DNS requests over TLS, or only the .global - in both cases it needs a TLS listener.